frontend/src/components/ProjectMemberPanel.vue

```diff
@@ -17,6 +17,10 @@
         class="ml-1 inline text-sm normal-link"
         @click.prevent="
           () => {
+            if (!has3rdPartyAuthFeature) {
+              state.showFeatureModal = true;
+              return;
+            }
```

Comment:
We'd better not write the complex logic into the `template`, move it in the `<script>`, keep <template> readable.

On the other hand, the function you named might reuse it somehow.